### ISSUE TITLE
Interpreting attention weights for more than one input features.

### ISSUE THREAD

**Issue by kwon3969** [philipperemy/keras-attention-mechanism#54]
How can we get attention weights for each input feature when our input consists of multiple inputs?
I am getting only one array of attention weights and I am not sure how to interpret it for multiple inputs.

shape of attention weights (attached as fig) is:
(300, 6) 
where 6 is the sequence_length/lookback steps/time steps. 

![attention_weight](https://user-images.githubusercontent.com/52854229/110059164-0877a000-7da7-11eb-84d1-4687600b6889.png)

**Comment by philipperemy** [comment]
@kwon3969 we consider inputs with shape `(batch_size, time_steps, input_dim)`. The attention weights (after softmax) have shape `(batch_size, time_steps)`.  The `input_dim` is no longer here. You can only know where the attention is on the `time_steps` axis. And not at the input level. This attention mechanism is too simple to give you this information. Does it answer your question?

**Comment by philipperemy** [comment]
Closing this question for inactivity.
